tmt-app/src/Components/NavBar.jsx

```diff
@@ -0,0 +1,171 @@
+  List,
+  ListItem,
+  ListItemButton,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+
+import BTLogo from '../Images/icons/BT_Logo_1.svg';
+
+function NavBar() {
+  let [navMenu, setNavMenu] = useState(false);
+
+  const handleOpenNavMenu = () => {
+    setNavMenu(true);
+  };
+  const handleCloseNavMenu = () => {
+    setNavMenu(false);
+  };
+  const tabs = [
+    {
+      name: 'Majors & Careers',
+      path: '/',
+    },
+    {
+      name: 'Team',
+      path: '/team',
+    },
+    {
+      name: 'Take the Quiz',
+      path: '/quiz/classes',
+    },
+  ];
+  return (
+    <AppBar
+      sx={{
+        position: 'sticky',
+        backgroundColor: 'BTMilkTea.main',
+        marginBottom: '3.5rem',
+      }}
+    >
+      <Container maxWidth="xl">
+        <Toolbar sx={{ justifyContent: 'space-between' }}>
+          <Link key="Homepage" to="/" component={RouterLink}>
+            <img src={BTLogo} alt="logo" />
+          </Link>
+          <Box
+            sx={{
+              display: {
+                xs: 'flex',
+                md: 'none',
+              },
+            }}
+          >
+            <IconButton
+              aria-label="options"
+              color="black"
+              onClick={handleOpenNavMenu}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Drawer
+              anchor="right"
+              open={navMenu}
+              onClose={handleCloseNavMenu}
+              ModalProps={{ disableScrollLock: 'true' }}
+              sx={{
+                display: {
+                  xs: 'flex',
+                  md: 'none',
+                },
+              }}
+            >
+              <Box>
+                <ListItem sx={{ justifyContent: 'flex-end' }}>
+                  <IconButton
+                    aria-label="options"
+                    color="black"
+                    onClick={handleCloseNavMenu}
+                  >
+                    <CloseIcon />
+                  </IconButton>
+                </ListItem>
+                {tabs.map((tab, id) => {
+                  return (
+                    <List key={`${tab}-{id}`}>
+                      <ListItem disablePadding onClick={handleCloseNavMenu}>
+                        <ListItemButton>
+                          <Link
+                            key={id}
+                            to={tab.path}
+                            component={RouterLink}
+                            underline="none"
+                            sx={{
+                              margin: '0 1rem',
+                              color: 'Boba.main',
+                            }}
+                          >
+                            <Typography variant="h5">{tab.name}</Typography>
+                          </Link>
+                        </ListItemButton>
+                      </ListItem>
+                    </List>
```

Comment:
btw why do we need a whole new list item here for every tab?

Reply:
not too sure why I did that haha... I will fix it